Poll on the relationship between JSPI and core-stacks (#1775)
WebAssembly/meetings: stack/2025/stack-2025-03-10.md

## Agenda items

1. Ask me anything about JSPI. Discussion led by Francis McCabe
2. Informal poll on the relationship between JSPI and core-stacks by Keith Miller (15m)

>     1. "Does JSPI's existance preclude a core-stack switching design is not side-stack based"

Reviewer: ```suggestion
    1. "Does JSPI's existence require that any future core-stack switching design is side-stack based?"
```
